tests/integration/steps/integration_steps.go

```diff
@@ -145,26 +146,21 @@ func deployCluster(t *testing.T, namespace, customValues string, helmValues map[
 	WaitForCassDcToBeReady(t, namespace)
 }
 
-func DeployClusterWithValues(t *testing.T, namespace, options, customValues string, nodes int, upgrade bool, useLocalCharts bool, version string) {
-	log.Printf("Deploying a cluster with %s options using the %s values", options, customValues)
+func DeployClusterWithValues(t *testing.T, namespace, medusaBackend, customValues string, nodes int, upgrade bool, useLocalCharts bool, version string) {
+	log.Printf("Deploying a cluster with %s Medusa backend using the %s values", medusaBackend, customValues)
 
 	helmValues := map[string]string{}
-	if options == "default" {
-		helmValues = map[string]string{
-			"reaper.ingress.host": "repair.127.0.0.1.nip.io",
-		}
-	}
-	if options == "minio" {
+
+	switch strings.ToLower(medusaBackend) {
+	case "minio":
 		serviceName := MinioServiceName(t)
-		helmValues = map[string]string{
-			"medusa.storage_properties.host": fmt.Sprintf("%s.minio.svc.cluster.local", serviceName),
+		helmValues["medusa.storage_properties.host"] = fmt.Sprintf("%s.minio.svc.cluster.local", serviceName)
+	case "s3":
```

Comment:
```suggestion
	case "s3", "azure_blobs", "google_storage":
```

We'd need to make the bucket name configurable for other storage backends as well if we want to run CI on our forks, or even locally on our laptops.

Reply:
Thanks for spotting this – I wonder how the tests are passing without this change... And indeed we might look into making the bucket name parameterizable. It's not required for the Azure tests though, you just need to create a container named like your S3 bucket.

Reply:
Once you change the case instruction to include azure, it will be parameterized through the `K8SSANDRA_MEDUSA_BUCKET_NAME` env variable, just like S3.
I guess the tests are skipped in the PR checks because of the missing secrets and in your fork you may be using `k8ssandra-medusa` as bucket name, which is the name used by default in the integration tests.